### ISSUE TITLE
meaning of audio_value and audio_id

### ISSUE THREAD

**Issue by jsalsman** [SND96/MTurk-TransciptionTask#1]
@SND96 on [lines 92-3 of mturk_hit.html,](master/mturk_hit.html#L92)

    audio_value = 1; //Given value depending on which audio recording is being listened to.

    if (audio_id != "") {

        alert("You have already listened to this audio file. Try some other one");

I just want to make sure we understand that the point of this is not (only) to keep the turker from listening to the same particular recording, but to keep them from listening to any other recording of the same prompting phrase. 

For example, if a turker hears, "what's your name," pronounced correctly, and transcribes it as such, we want to make sure they don't hear any more instances of attempts to pronounce that same phrase. Because, for example, if they next hear, "hats are nam," they might remember the first time they heard the correct pronunciation and be more likely to try to type it in the right way even if they wouldn't have understood the mispronunciations if they heard the poorly pronounced version first.

I understand that this is just placeholder code, but I wanted to clear that up just in case.

It might make sense to replace those audio_value and audio_id identifiers with something else, like prompting_phrase_id and attempted_phrases, and re-work the placeholder code to show adding the prompting_phrase_id to the attempted_phrases cookie. 

But that brings up another question: how long is a cookie allowed to be? If a transcriptionist transcribes 100 phrases, will the 101'th id run out of room in the cookie?  If so, do we need to just store a generic userid of some sort in the cookie, and rely on a server-side database to make sure that the same turker with multiple logins isn't listening to duplicate prompting_phrase_id's?

**Comment by SND96** [comment]
Yeah I wasn't sure on what data we'll be providing for the turkers so I left it pretty basic for now. 
Maybe we can encode part of the id according to the phrase being uttered so that we can still have different ID's for each recording.

We can use the Mechanical Turk API to connect to the server and store the cookies if needed and compare them. If its possible, different user ID's can be clubbed together and treated as one if a user tries to use multiple accounts. This should remove the need for storing the cookies on the user's browser and instead make use of MTurk API to accept and reject them for tasks.
